### ISSUE TITLE
Remove broken feature that auto-updates job status

### ISSUE THREAD

**Issue by ericfranz** [OSC/osc_machete_rails#62]
This hasn't worked reliably:

https://github.com/OSC/osc_machete_rails/blob/48f16f9fe5d1d5f3e315d0281fca830995db5425/lib/osc_machete_rails/statusable.rb#L63-L80

and

https://github.com/OSC/osc_machete_rails/blob/48f16f9fe5d1d5f3e315d0281fca830995db5425/lib/osc_machete_rails/engine.rb#L11-L22

Instead, update the generators to do this, and recommend for existing apps in the app upgrade to do this:

```diff
 class SimulationsController < ApplicationController
   before_action :set_simulation, only: [:show, :edit, :update, :destroy, :submit, :copy]
+  before_action :update_jobs, only: [:index, :show]


   private
+    def update_jobs
+      SimulationJob.active.to_a.each(&:update_status!)
+    end
 end
```

**Comment by ericfranz** [comment]
@basilgohar @nickjer @brianmcmichael 

I instruct developers porting apps to do this:

https://github.com/AweSim-OSC/documentation/wiki/Port-an-existing-app-to-OOD#7-explicitly-update-status-of-jobs-on-each-request

But newly generated apps will have not have these changes. Maybe we should push this change out soon.

**Comment by nickjer** [comment]
As long as you are careful about order dependence. In fact I typically run the `set_simulation` method in each of those controller actions. That way I am sure the `update_jobs` before_action is the first one called as there are no other `before_action`'s

**Comment by nickjer** [comment]
See http://craftingruby.com/posts/2015/05/31/dont-use-before-action-to-load-data.html

**Comment by ericfranz** [comment]
Good call. I updated the directions on the wiki https://github.com/AweSim-OSC/documentation/wiki/Port-an-existing-app-to-OOD#7-explicitly-update-status-of-jobs-on-each-request
